client/src/js/services/grid/TransactionService.js

```diff
@@ -295,6 +295,26 @@ function TransactionService($timeout, util, uiGridConstants, bhConstants, Notify
     return true;
   };
 
+
+  /**
+   * @method validateTransaction
+   *
+   * @description
+   * This method is used to check if a transaction is balanced by their record
+   */
+  function validateTransaction(entity) {
+    var dataEntity = entity.data.data;
```

Comment:
A better name for this is `transaction`, since we are editing a transaction and the function is called `validateTransaction()`.